metanorm/normalizers/ftp.py

```diff
@@ -0,0 +1,146 @@
+                elif "weeks" in ftp_add_and_file_name.split('/'):
+                    return starting_date+relativedelta(days=+6)  # end time is six days after start day
+                elif file_name[10] == 'v':  # file is a month file.So,the end time must be  end of month
+                    return starting_date + relativedelta(
+                        days=calendar.monthrange(starting_date.year, starting_date.month)[1]-1)
+                else:
+                    return starting_date
+            elif raw_attributes['url'].startswith('ftp://anon-ftp.ceda.ac.uk/neodc/esacci/sst/data/CDR_v2/Climatology/L4/v2.1'):
+                return dateutil.parser.parse("20100101T000000").replace(tzinfo=tzutc())\
+                    + relativedelta(days=+int(file_name[1:4]))
+            elif raw_attributes['url'].startswith("ftp://ftp.gportal.jaxa.jp"):
+                # time portion of the filename is extracted and then parsed with "dateutil.parser.parse"
+                return self.get_time_coverage_start(raw_attributes)
+
+    def get_provider(self, raw_attributes):
+        """ returns the suitable provider based on the filename """
+        if 'url' in raw_attributes:
+            urls_provider = {'ftp://anon-ftp.ceda.ac.uk/neodc/esacci/sst/data/CDR_v2/Climatology/L4/v2.1': 'ESA/CCI',
+                             "ftp://ftp.remss.com/gmi/": 'NASA/GSFC/SED/ESD/LA/GPM',
+                             "ftp://ftp.gportal.jaxa.jp/standard": 'JP/JAXA/EOC'}
+            return self.looper(urls_provider, raw_attributes['url'], "get_gcmd_provider")
+
+    def get_dataset_parameters(self, raw_attributes):
+        """ DANGER!!!!! return list with different parameter from wkv variable """
+        if 'url' in raw_attributes:
+            urls_dsp = {'ftp://anon-ftp.ceda.ac.uk/neodc/esacci/sst/data/CDR_v2/Climatology/L4/v2.1': ['sea_surface_temperature'],
+                        'ftp://ftp.gportal.jaxa.jp/standard/GCOM-W/GCOM-W.AMSR2': ['sea_surface_temperature'],
+                        "ftp://ftp.remss.com/gmi/": ['wind_speed', 'atmosphere_mass_content_of_water_vapor',
+                                                     'atmosphere_mass_content_of_cloud_liquid_water', 'rainfall_rate'], }
+            for url in urls_dsp.keys():
+                if raw_attributes['url'].startswith(url):
+                    return [pti.get_cf_standard_name(cf_parameter) for cf_parameter in urls_dsp[url]]
+        return []
+
+    def get_location_geometry(self, raw_attributes):
+        """ DANGER!!!!! jaxa remains(others must be checked!)"""
+        if 'url' in raw_attributes:
+            urls_geometry = {'ftp://anon-ftp.ceda.ac.uk/neodc/esacci/sst/data/CDR_v2/Climatology/L4/v2.1': GEOSGeometry(('POLYGON((-180 -90, -180 90, 180 90, 180 -90, -180 -90))'), srid=4326),
+                             'ftp://ftp.gportal.jaxa.jp/standard/GCOM-W/GCOM-W.AMSR2': GEOSGeometry(('POLYGON((1 1, 1 2, 2 2, 2 1, 1 1))'), srid=4326),
```

Comment:
Let's fix it for L3 SST so far.
```suggestion
                             'ftp://ftp.gportal.jaxa.jp/standard/GCOM-W/GCOM-W.AMSR2/L3.SST_10': GEOSGeometry(('POLYGON((-180 -90, -180 90, 180 90, 180 -90, -180 -90))'), srid=4326),
                             'ftp://ftp.gportal.jaxa.jp/standard/GCOM-W/GCOM-W.AMSR2/L3.SST_25': GEOSGeometry(('POLYGON((-180 -90, -180 90, 180 90, 180 -90, -180 -90))'), srid=4326),
```